### PR TITLE
#90 pip install frontend

### PR DESCRIPTION
**Pull request by tomster**

### CODE REVIEW

**Comment by tomster** [comment]
actually, this would be a good opportunity to rename the folder from the generic `htdocs` perhaps to `senic_hub_frontend`?

**Comment by grunskis** [comment]
LGTM! Do the rename as well then ;)

**Comment by grunskis** [comment]
👍

File: deployment/roles/hub-ubuntu/defaults/main.yml [hunk 1]
```diff
@@ -3,5 +3,5 @@ ploy_adhoc_channel: 1
 wwwuser: www-data
 hostname: "{{ploy_hostname}}-{{ansible_machine_id[:4]}}"
 backend_deploy_location: /srv/senic_hub
-frontend_document_root: /srv/senic_hub/venv/htdocs
```
Comment (larsblumberg) [inline deployment/roles/hub-ubuntu/defaults/main.yml]: the `venv` folder is located inside the `senic_hub` folder. Why do you replicate this prefix in `senic-hub-htdocs`?

File: setup.py [hunk 2]
```diff
@@ -35,9 +35,10 @@ def find_data(top):
     package_data={
         name: [
             '.coveragerc',
-            'backend/tests/*.py',
-            'backend/tests/data/*.*',
-            'backend/views/*.*',
+            'senic_hub/backend/tests/*.py',
+            'senic_hub/backend/tests/data/*.*',
+            'senic_hub/backend/views/*.*',
+            'senic_hub/frontend/distribution',
```
Comment (larsblumberg) [inline setup.py]: When we run `npm run build` the frontend output is put into the `build` folder, not `distribution`. Can we unify the names?

Comment (tomster) [inline setup.py]: unfortunately both requirements are hardcoded by setuptools and npm respectively, the only working solution i found was to rename via the Makefile

File: deployment/roles/hub-ubuntu/defaults/main.yml (hunk 1 above)
Comment (larsblumberg) [inline deployment/roles/hub-ubuntu/defaults/main.yml]: As discussed orally, `pip install` can be done into any `venv`, it mustn't necessarily be inside our `senic_hub` folder.

File: setup.py (hunk 2 above)
Comment (larsblumberg) [inline setup.py]: Got it, `build` is a reserved name

**Pull request merged**